### PR TITLE
feat(postgresql/docker): Added docker compose to initialize an instance of Postgres

### PR DESCRIPTION
**Pull request by miguel-merlin**
#32

### CODE REVIEW

**Review by JonMike8 — APPROVED** [review]

File: docker-compose.yaml
Comment (ericzty) [inline docker-compose.yaml:1]: why are we exposing the postgres port here? If we put the backend in the same network then we don't have to expose it.

POSTGRES_USER, POSTGRES_PASSWORD should be env vars and not preset in the docker-compose.
RE: POSTGRES_DB, let's just name it nycmesh

**Review by ericzty — CHANGES_REQUESTED** [review]

File: docker-compose.yaml
Comment (miguel-merlin) [reply docker-compose.yaml:1]: Just pushed the changes!

**Comment by ericzty** [comment]
Relates to #15

**Comment by ericzty** [comment]
Could you push a .env.example with example env files.
the rest looks good

**Comment by miguel-merlin** [comment]
> Could you push a .env.example with example env files. the rest looks good

Done!

**Review by ericzty — APPROVED** [review]

**Pull request merged**